api/src/services/healthService.js

```diff
@@ -203,19 +141,20 @@ async function checkBlockchainBlocksFinalization(req, res) {
 
 async function checkBlockchainBlocksProducing(req, res) {
   const { query } = req;
-  const networks = splitParams(query.networks);
+  let networks = splitParams(query.networks);
   const cereConfig = blockchains.find(blockchain => blockchain.name === blockchainNames.CERE);
-  let errors = [];
-
-  try {
-    validateNoEmptyFields({networks});
-    validateNetworksNames(networks);
-  } catch (error) {
-    return res.status(400).json({
-      msg: error.message,
-    });
+  
+  if (networks) {
+    const validationErrors = validateNetworks(networks);
+    if (validationErrors.length) {
+      return res.status(400).json({msg: validationErrors });
```

Comment:
msg -> "X failed"
errors -> list of errors

Reply:
fixed